sql/thriftserver/README.md

```diff
@@ -0,0 +1,3 @@
+Thrift commands to generate files from TCLIService.thrift:
+--------------------
+thrift --gen java:beans,hashcode -o src/gen/thrift if/TCLIService.thrift
```

Comment:
Could you add `generated_annotations=undated` to avoid diffs?
And add a sentence like `Please use Thrift 0.9.3 available from https://www.apache.org/dist/thrift/0.9.3/.`

Reply:
> Could you add `generated_annotations=undated` to avoid diffs?
> And add a sentence like `Please use Thrift 0.9.3 available from https://www.apache.org/dist/thrift/0.9.3/.`

Sorry for didn't notice this point.  I'll update this later.